cmd/influxd/run/config_command.go

```diff
@@ -74,6 +76,12 @@ func (cmd *PrintConfigCommand) parseConfig(path string) (*Config, error) {
 	if _, err := toml.DecodeFile(path, &config); err != nil {
 		return nil, err
 	}
+	if len(config.UDPs) == 0 {
```

Comment:
@otoolep  Because 4 lines above, there is ```config := NewConfig()``` where ```[]udp.Config{udp.NewConfig()``` was called before this workaroud.
But, I'm agree with you, It seems to me kind of useless here, but I was not sure... so I keep it.
Do you think I can delete it ?